sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StreamingWriteFn.java

```diff
@@ -80,6 +89,20 @@
     this.toTableRow = toTableRow;
   }
 
+  @Setup
+  public void setup() {
+    // record latency upto 30 seconds in the resolution of 20ms
+    histogram = Histogram.of(0, 30000, 1500, true);
```

Comment:
30s seems a bit small. What is the timeout set to? I recommend making the maximum at least another 30s or so larger than the timeout.

Reply:
Done.